test/unit/connectors/EmailConnectorSpec.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import akka.actor.ActorSystem
+import com.github.tomakehurst.wiremock.client.WireMock._
+import com.github.tomakehurst.wiremock.matching.EqualToJsonPattern
+import config.FrontendAppConfig
+import models.{ApplicationSubmittedEmail, ApplicationSubmittedParameters}
+import org.apache.http.HttpStatus
+import org.mockito.BDDMockito._
+import org.scalatest.mockito.MockitoSugar
+import play.api.Environment
+import play.api.libs.ws.WSClient
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.audit.DefaultAuditConnector
+import uk.gov.hmrc.play.bootstrap.http.DefaultHttpClient
+import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
+
+class EmailConnectorSpec extends UnitSpec
```

Comment:
This needs refactoring (see #83)

We will need this class to extend `TestConnector`

Reply:
I'm re-factoring that.  Do you mean `ConnectorTest`?